File: java/Basic/ApiKey/src/main/java/ApiKey.java
import com.factset.sdk.PAEngine.ApiClient;
import com.factset.sdk.PAEngine.ApiException;
import com.factset.sdk.PAEngine.api.FrequenciesApi;
import com.factset.sdk.PAEngine.auth.HttpBasicAuth;
import com.factset.sdk.PAEngine.models.FrequencyRoot;

public class ApiKey {
    public static void main(String[] args) {
        ApiClient apiClient = new ApiClient();

        HttpBasicAuth FactSetApiKey = (HttpBasicAuth) apiClient.getAuthentication("FactSetApiKey");
        FactSetApiKey.setUsername("USERNAME-SERIAL");
        FactSetApiKey.setPassword("API-KEY");

        FrequenciesApi frequenciesApi = new FrequenciesApi(apiClient);

        try {
            FrequencyRoot frequencyRoot = frequenciesApi.getPAFrequencies();
            System.out.println(frequencyRoot.getData());
        } catch (ApiException e) {
            e.printStackTrace();
        }
    }
}

File: dotnet/Basic/ApiKey/ApiKey.cs
using System;
using FactSet.SDK.PAEngine.Api;
using FactSet.SDK.PAEngine.Model;
using System.Threading;
using System.Threading.Tasks;
using System.Collections.Generic;

namespace Factset.SDK.Sample.Basic.ApiKey
{
    class ApiKey
    {
        static readonly CancellationTokenSource cts = new CancellationTokenSource();

        static async Task Main(string[] args)
        {
            FactSet.SDK.PAEngine.Client.Configuration configuration = new FactSet.SDK.PAEngine.Client.Configuration();
            configuration.Username = "USERNAME-SERIAL";
            configuration.Password = "API-KEY";

            FrequenciesApi frequenciesApi = new FrequenciesApi(configuration);

            try
            {
                cts.CancelAfter(5000);

                FrequencyRoot frequencyRoot = await frequenciesApi.GetPAFrequenciesAsync(cts.Token);

                foreach (var pair in frequencyRoot.Data)
                {
                    Console.WriteLine($"{pair.Key} : {pair.Value}");
                }
            }
            catch (OperationCanceledException)
            {
                Console.WriteLine("\nTasks cancelled: timed out.\n");
            }
            finally
            {
                cts.Dispose();
            }
        }
    }
}
